support-frontend/assets/helpers/productPrice/fulfilmentOptions.js

```diff
@@ -6,6 +6,7 @@ import { countryGroups } from 'helpers/internationalisation/countryGroup';
 
 const HomeDelivery: 'HomeDelivery' = 'HomeDelivery';
 const Collection: 'Collection' = 'Collection';
+const DigitalVoucher: 'DigitalVoucher' = 'DigitalVoucher';
```

Comment:
After thinking about this a bit more, I think it's better if we stick with HomeDelivery and Collection as the two fulfilment options for papers. I think it might be better if the subscription card text and the paper vouchers text are subsets of Collection.

Reply:
I think you're right; I've removed the DigitalVoucher type and reworked the way we're getting the 'What next' text so the specific digital voucher copy sits as an option under Collection.